src/main/java/io/kokuwa/maven/helm/AbstractHelmMojo.java

```diff
@@ -231,7 +235,7 @@ void callCli(String command, String errorMessage, final boolean verbose, String
 			p.waitFor();
 			exitValue = p.exitValue();
 		} catch (Exception e) {
-			throw new MojoExecutionException("Error processing command [" + replaceK8sToken(command) + "]", e);
+			throw new MojoExecutionException("Error processing command [" + command + "]", e);
```

Comment:
`command` can contains sensitive data - should be visible in exception?

Reply:
meh, your right.